data/MLO/projects.yaml

```
@@ -490,3 +490,30 @@ projects:
       - PyTorch
     date_added: 2022-07-07
     date_updated: 2022-07-07
+
+  anomaly-detection:
+    name: Distributed Homomorphic Anomaly Detection
+    categories:
+      - Learning
+    applications:
+      - Info
+    description: Runs anomaly detection algorithms in a distributed and encrypted fashion.
```

As we're not doing any encryption here, the description should reflect that.
```suggestion
    description: Proof-of-concept ML algorithms to do anomaly detection compatible with distributed, encrypted algorithms.
```

---

Makes sense. added your suggestion.